pkg/route/ingress/ingress.go

```diff
@@ -423,6 +423,10 @@ func (c *Controller) sync(key queueKey) error {
 			continue
 		}
 		for _, path := range rule.HTTP.Paths {
+			pathValue := path.Path
+			if ingress.Annotations["route.openshift.io/termination"] == "passthrough" && path.Path == "/" {
```

Comment:
Use `terminationPolicyAnnotationKey` and `routev1.TLSTerminationPassthrough`:
```suggestion
			if ingress.Annotations[terminationPolicyAnnotationKey] == string(routev1.TLSTerminationPassthrough) && path.Path == "/" {
```
I suggested before that we should ignore ingresses that specify a nonempty path (for values of "nonempty" that include `/`) and passthrough termination.  How does the logic you have here handle such ingresses?